src/main/java/edu/harvard/iq/dataverse/engine/command/impl/FinalizeDatasetPublicationCommand.java

```diff
@@ -170,12 +171,14 @@ private void publicizeExternalIdentifier(CommandContext ctxt) throws CommandExce
         String protocol = theDataset.getProtocol();
         IdServiceBean idServiceBean = IdServiceBean.getBean(protocol, ctxt);
         try {
-            idServiceBean.publicizeIdentifier(theDataset);
+        	//A false return value indicates a failure in calling the service
+            if(!idServiceBean.publicizeIdentifier(theDataset)) throw new Throwable();
```

Comment:
Shouldn't you be throwing something more specific? At the very least, and `Exception`

Reply:
The try/catch block catches Throwable and never looks at it's contents - it just creates a CommandException which is what is passed upward. I can change this to Exception (and the one on line 181), but I don't know if there is really a Throwable but non-Exception that can be thrown from something else to know if the catch can be changed.

Reply:
I wouldn't change the catch as you never know why it was put there.